tests/unit/cli/admin/test_import_targets.py

```diff
@@ -0,0 +1,541 @@
+                signed=pretend.stub(
+                    delegations=pretend.stub(succinct_roles=None)
+                )
+            )
+        )
+
+        with pytest.raises(import_targets.click.ClickException) as err:
+            import_targets._get_succinct_roles("http://127.0.0.1/metadata")
+        assert "Failed to get Targets succinct roles" in str(err)
+
+
+class TestImportTargetsGroupCLI:
+    def test_import_targets(self, client, test_context):
+        test_context["settings"].SERVER = "fake-server"
+        test_context["settings"].TOKEN = "test-token"
+
+        import_targets.is_logged = pretend.call_recorder(
+            lambda *a: pretend.stub(state=True)
+        )
+        import_targets.get_headers = pretend.call_recorder(
+            lambda *a: "headers"
+        )
+        fake_response = pretend.stub(
+            status_code=200,
+            json=pretend.call_recorder(lambda: {"data": {"bootstrap": True}}),
+        )
+        import_targets.request_server = pretend.call_recorder(
+            lambda *a, **kw: fake_response
+        )
+        import_targets._get_succinct_roles = pretend.call_recorder(
+            lambda *a: "fake_succinct_roles"
+        )
+        import_targets.create_engine = pretend.call_recorder(
+            lambda *a: pretend.stub(
+                connect=pretend.call_recorder(
+                    lambda: pretend.stub(
+                        commit=pretend.call_recorder(lambda: None)
+                    )
+                )
+            )
+        )
+        import_targets.Table = pretend.call_recorder(
+            lambda *a, **kw: "rstuf_table"
+        )
+        import_targets._check_csv_files = pretend.call_recorder(
+            lambda **kw: None
+        )
+        import_targets._import_csv_to_rstuf = pretend.call_recorder(
+            lambda *a: None
+        )
+        import_targets.publish_targets = pretend.call_recorder(
+            lambda *a: "fake_task_id"
+        )
+        import_targets.task_status = pretend.call_recorder(
+            lambda *a: {"status": "SUCCESS"}
+        )
+
+        options = [
+            "-metadata-url",
+            "http://127.0.0.1/metadata/",
+            "-db-uri",
+            "postgresql://postgres:secret@127.0.0.1:5433",
+            "-csv",
+            "targets1of2.csv",
+            "-csv",
+            "targets2of2.csv",
+        ]
+        result = client.invoke(
+            import_targets.import_targets, options, obj=test_context
+        )
+        assert result.exit_code == 0, result.output
+        assert "Finished." in result.output
+        assert import_targets.is_logged.calls == [
+            pretend.call("fake-server", "test-token")
+        ]
+        import_targets.request_server.calls == [
+            pretend.call(
+                "fake-server",
+                import_targets.URL.bootstrap.value,
+                import_targets.Methods.get,
+                headers="headers",
```

Comment:
The pretend will record the call based on your function implementation. If the code implementation used a keyword to call it would be recorded as is.
https://github.com/vmware/repository-service-tuf-cli/blob/73b0c9c3e242c0c8faf10fcea5f32eabe1bfab4e/repository_service_tuf/cli/admin/import_targets.py#L160
To assert, you must use the same record. If you change the implementation not to use the keyword call, you will also catch it in the assets.